debug_module.tex

```diff
@@ -179,15 +179,18 @@ \section{Run Control} \label{runcontrol}
 For every hart, the Debug Module tracks 4 conceptual bits of state: halt
 request, resume ack, halt-on-reset request,  and hart reset.
 (The hart reset and halt-on-reset request bits are optional.)
-These bits all reset to 0. A debugger can write them for the currently selected
-harts through \Fhaltreq, \Fsetresethaltreq/\Fclrresethaltreq and
-\Fhartreset in \Rdmcontrol.
-In addition the DM receives halted and running signals from each hart.
-
+These 4 bits reset to 0, except for resume ack, which may reset to either 0 or 1.
+The DM receives halted, running, and havereset signals from each hart.
+The debugger can observe the state of resume ack in \Fallresumeack and
+\Fanyresumeack, and the state of halted, running, and havereset signals
```

Comment:
Why th two separate lists?

Reply:
Observing versus writing? Which two lists exactly are you referring to?